bytecode/tests/test_concrete.py

```diff
@@ -813,6 +813,36 @@ def test_extreme_compute_jumps_convergence(self):
 
         code.to_code(compute_jumps_passes=(len(labels) + 1))
 
+    def test_general_constants(self):
+        """Test if general object could be linked as constants.
+
+        """
+        class CustomObject:
+            pass
+
+        class UnHashableCustomObject:
+            def __eq__(self, other):
+                return self is other  # pragma: no cover
```

Comment:
This is the default behavior of `__eq__`.

Wouldn't be easier to make a class unhashable by defining `__hash__ = None`?